src/codeflare_sdk/__init__.py

```diff
@@ -19,3 +19,13 @@
 from .job import RayJobClient
 
 from .utils import generate_cert
+
+from importlib.metadata import version, PackageNotFoundError
+
+try:
+    __version__ = version(
+        "codeflare-sdk"
```

Comment:
One other comment. I think:

```
__package__ or __name__
```
would also work here

Reply:
Won't this change the whole idea of getting the version details with codeflare_sdk.__version__ ??
If this is what needs to be used, I will make the change.

Reply:
Sorry, I think I wasn't quite clear. I meant:
`("codeflare-sdk")` -> `(__package__ or __name__)`, so just dynamically getting the package name rather than hard coding it.

Reply:
We can change it in the future if we need to for some reason. I'm happy to have this merged in. Thanks for the contribution @snalawad